app/server/makeAdmin.js

```diff
@@ -0,0 +1,47 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+import { check } from 'meteor/check';
+
+
+Accounts.onCreateUser((options, newUser) => {
+    const anyUser = Meteor.users.find({}).count();
+
+    if (anyUser > 0) {
+        newUser.isAdmin = false;
+    } else {
+        newUser.isAdmin = true;
+    }
+
+    return newUser;
+});
+
+Meteor.methods({
+    setAdmin(userId) {
+        check(userId, String);
+
+        const admin = Meteor.user().isAdmin;
+
+        if (admin) {
+            const doc = Meteor.users.findOne({ _id: userId });
```

Comment:
`const user = ...` 
And also if you find by id, you can just write `Collection.findOne(stringId)`, it works also in updates etc.